docs/content/en/docs/tasks/cluster/cluster-upgrades/vsphere-and-cloudstack-upgrades.md

```diff
@@ -47,6 +47,8 @@ version.
 Irrespective of a Kubernetes version change, the upgrade command will always upgrade the internal EKS
 Anywhere components mentioned above to their latest available versions. All upgrade changes are backwards compatible.
 
+Specifically for Snow provider, see [Upgrade EKS Anywhere AMIs in Snowball Edge devices](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) to upgrade and use a new Admin instance which contains a newer version of EKS Anywhere CLI binary.
```

Comment:
Could you be more specific about the new Snow Admin instances only being needed when upgrading to new versions of EKS Anywhere and that upgrades of other components can be done as described in this document?

Reply:
done